cloud-init/copr_test

```diff
@@ -168,8 +168,8 @@ EOF
     lxc file push "$TEMP_D/$repo" "$name/etc/yum.repos.d/$repo" ||
         fail "failed: setting up copr.repo"
 
-    install_pkgs "$name" epel-release ||
-        fail "failed to install epel-release"
+    install_pkgs "$name" epel-release epel-next-release ||
```

Comment:
We might need to encode packages in `jenkins-jobs/cloud-init/centos.yaml` eventually, since the current Stream 9 installs directly from an http address[1]. Since `python3-httpretty` is not yet available for CentOS Stream 9, however, we can just "wait and see". Maybe that won't be needed when our dependencies are sorted out in EPEL 9.

[1] https://docs.fedoraproject.org/en-US/epel/#_el9